backend/typescript/models/user.model.ts

```diff
@@ -20,4 +23,7 @@ export default class User extends Model {
 
   @Column({ type: DataType.BOOLEAN })
   active!: boolean;
+
+  @HasMany(() => Review)
+  reviews!: Review[];
```

Comment:
Not too familiar with Sequalize syntax tbf but just wanted to confirm that you don't need a @BelongsToOne or something like that inside review.modal.ts to define the inverse relationship? 

I saw that you added the foreign key to the User table so that might be enough. Just thought of bringing it up in case you forgot to define the inverse relationship :)